CONTRIBUTING.md

```diff
@@ -29,24 +29,41 @@ Installing these requirements differs on each system.
 
 ### Instructions
 
-Get the code
-
+1. Make sure you have created your `GOPATH` directory (default to `$HOME/go`). You can also set a different path, like:
 ```sh
-git clone https://github.com/coyim/coyim.git
-cd coyim
+export GOPATH=$HOME/code
```

Comment:
maybe something like: `export GOPATH=$HOME/your-name` might be clearer.

Reply:
Can I ask why is this useful? I'm against explaining people how to setup their language environment: it changes frequently and it's the entry-level knowledge required to contribute.

The particular case of gopath is well documented and dont really is mandatory to have an exported 'GOPATH` variable (I dont):

```
$ go help gopath
The Go path is used to resolve import statements.
It is implemented by and documented in the go/build package.

The GOPATH environment variable lists places to look for Go code.
On Unix, the value is a colon-separated string.
On Windows, the value is a semicolon-separated string.
On Plan 9, the value is a list.

If the environment variable is unset, GOPATH defaults
to a subdirectory named "go" in the user's home directory
($HOME/go on Unix, %USERPROFILE%\go on Windows),
unless that directory holds a Go distribution.
Run "go env GOPATH" to see the current GOPATH.

See https://golang.org/wiki/SettingGOPATH to set a custom GOPATH.
```

Is there anything in our project that mandates a particular setup that diverges from the language default environment? If so, we document. Otherwise, we simply outline the workflow here.

Reply:
I believe the problem is really that our Makefile is "broken". `go get github.com/coyim/coyim` should just work but it doesn't.

Reply:
hey @juniorz, I agree with you that environment setups change frequently and don't necessary should be in projects instructions.

Do you think the last commit, which states just a reminder for GOPATH and PATH, is OK or should be removed?